echidna/core/root_help.py

```diff
@@ -0,0 +1,11 @@
+''' Helper script to allow --help option to work with ROOT import.
```

Comment:
Can you write it as ``... allow :option:`--help` option...`` please, so this shows up nicer in the build documentation